### ISSUE TITLE
Sencha .js files in project generate console error messages

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#2]
```
What steps will reproduce the problem?
1. Create a project that includes Sencha Touch


What is the expected output? What do you see instead?

There will be a stream of the following on the console:
IMPLEMENT resolveForAllocation for class 
org.eclipse.wst.jsdt.internal.compiler.ast.MessageSend


The root cause seems to be the JSDT parser barfing on the construct x = new 
(z()) Filed JSDT bug https://bugs.eclipse.org/bugs/show_bug.cgi?id=333387 


```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 5 Jan 2011 at 6:49

**Comment by GoogleCodeExporter** [comment]
```
I've submitted a fix to JSDT and will make sure it rolls into my next release.  
Still may need to incorporate building a patched JSDT.
```

Original comment by `paul.beu...@gmail.com` on 24 Jan 2011 at 5:48
- Changed state: **Started**
- Added labels: **Milestone-Release1.2.0**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with 
http://code.google.com/a/eclipselabs.org/p/mobile-web-development-with-phonegap/
source/detail?r=63
```

Original comment by `paul.beu...@gmail.com` on 20 Feb 2011 at 5:11

**Comment by GoogleCodeExporter** [comment]
```
Fixed with 1.2 release
```

Original comment by `paul.beu...@gmail.com` on 23 Feb 2011 at 4:24
- Changed state: **Fixed**
